clients/erigon-gnosis/mapper.jq

```diff
@@ -30,13 +30,18 @@ def to_bool:
 
 # Replace config in input.
 . + {
-   "auRaSeal": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+   "seal": {
+    "authorityRound": {
+      "step": "0x0",
+      "signature": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+    }
+  },
   "gasLimit": "0x989680",
   "timestamp": 0,
-  "coinbase": "0x0000000000000000000000000000000000000000",
+  # "coinbase": "0x0000000000000000000000000000000000000000",
   "baseFeePerGas": "0x3b9aca00",
-  "nonce": "0x0000000000000000",
-  "difficulty": "0x64",
+  # "nonce": "0x0000000000000000",
```

Comment:
Should the commented-out lines still be there?

Reply:
I leave it for now since it is unclear do we need it for previous (e.g. cacnun tests). Will remove after fixing all the tests